### PR TITLE
feature/EVS-GLWU_plots_initialized

### PR DESCRIPTION
**Pull request by SamiraArdani-NOAA**
<b>Note to developers: You must use this PR template!</b>

## Description of Changes

> Please include a summary of the changes and the related GitHub issue(s). Please also include relevant motivation and context.

This PR initializes the EVS-GLWU verification plots.  

## Developer Questions and Checklist
* Is this a high priority PR? If so, why and is there a date it needs to be merged by?
No.
* Do you have any planned upcoming annual leave/PTO?
No.
* Are there any changes needed for when the jobs are supposed to run?
No.  
- [x] The code changes follow [NCO's EE2 Standards](https://www.nco.ncep.noaa.gov/idsb/implementation_standards/ImplementationStandards.v11.0.0.pdf).
- [x] Developer's name is removed throughout the code and have used `${USER}` where necessary throughout the code.
- [x] References the feature branch for `HOMEevs` are removed from the code.
- [x] J-Job environment variables, COMIN and COMOUT directories, and output follow what has been [defined](https://docs.google.com/document/d/1JWg_4q80aYmmAoD21GFjp9R9y5-3w7WGM3-0HJk0Pjs/edit#heading=h.7ysbr191vzu4) for EVS.
- [x] Jobs over 15 minutes in runtime have restart capability.
- [x] If applicable, changes in the `dev/drivers/scripts` or `dev/modulefiles` have been made in the corresponding `ecf/scripts` and `ecf/defs/evs-nco.def`? 
- [x] Jobs contain the appropriate file checking and don't run METplus for any missing data.
- [x] Code is using METplus wrappers structure and not calling MET executables directly.
- [ ] Log is free of any ERRORs or WARNINGs.
There might be WARNINGs due to missing data.
## Testing Instructions

> Please include testing instructions for the PR assignee. Include all relevant input datasets needed to run the tests.

To test this PR, you need to run the driver script for stats and plots steps. 
2- Clone my fork in your local and checkout branch feature/EVS-GLWU_plots_initialized.
3- ln -s fix directory.
4- In COMIN, replace the ${USER} with emc.vpppg.
5- The stats step is just tested to see if the MPR line_type is added. Run the driver scripts for stats step located at: EVS/dev/drivers/scripts/stats/glwu/*
6- Run the driver scripts for plots step located at: EVS/dev/drivers/scripts/plots/glwu/*

### CODE REVIEW

File: ecf/scripts/plots/glwu/jevs_glwu_wave_grid2obs_plots.ecf
Comment (malloryprow) [inline ecf/scripts/plots/glwu/jevs_glwu_wave_grid2obs_plots.ecf:1]: This has `export OBTYPE=NDBC` but dev/drivers/scripts/plots/glwu/jevs_glwu_wave_grid2obs_plots.sh has `export OBTYPE=NDBC_STANDARD`.

Additionally this has `select=2` and dev/drivers/scripts/plots/glwu/jevs_glwu_wave_grid2obs_plots.sh has `select=1`.

**Comment by malloryprow** [comment]
I will kick off the stats job at 1500Z, when the glwu stats job runs.

**Comment by malloryprow** [comment]
### jevs_glwu_wave_grid2obs_stats.sh

**Log File:** /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr592/EVS/dev/drivers/scripts/stats/glwu/jevs_glwu_wave_grid2obs_stats.o158976459
**COMOUT:** /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr592/evs/v2.0/stats/glwu
**DATA:** /lfs/h2/emc/stmp/mallory.row/evs_test/prod/tmp/jevs_glwu_wave_grid2obs_stats.158976459.cbqs01

**Comment by SamiraArdani-NOAA** [comment]
@malloryprow, I checked the .o file. No ERROR was found. Also, the Point-Stat files now show the MPR line_type.

**Comment by malloryprow** [comment]
### jevs_glwu_wave_grid2obs_plots.sh
**Log File:** /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr592/EVS/dev/drivers/scripts/plots/glwu/jevs_glwu_wave_grid2obs_plots.o158977139
**COMOUT:** /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr592/evs/v2.0
**DATA:** /lfs/h2/emc/stmp/mallory.row/evs_test/prod/tmp/jevs_glwu_wave_grid2obs_plots.158977139.cbqs01

**Comment by SamiraArdani-NOAA** [comment]
@malloryprow, I see no ERRORs in the .o file. I see that the plots are now generated for both last 31days and last90days and the tar files contain both type of plots: time_series and lead_average.

**Review by malloryprow — APPROVED** [review]
Code changes are good and testing successful.

**Comment by malloryprow** [comment]
@AndrewBenjamin-NOAA This needs your review (and pending approval) when you get a chance.

**Comment by AndrewBenjamin-NOAA** [comment]
@SamiraArdani-NOAA, have you untarred and taken a look at your plots? Do you have a location where I would be able to take a look?

**Comment by SamiraArdani-NOAA** [comment]
@AndrewBenjamin-NOAA, Yes, I have untarred them in EMCRZDM at: /home/people/emc/www/htdocs/users/verification/ocean_lake/glwu/test/grid2obs/images/*. Also, I have created a /test webpage at: https://www.emc.ncep.noaa.gov/users/verification/ocean_lake/glwu/test/grid2obs/wsp.
I still need to enhance webpage dropdowns functionality. This verification contains significant wave height (htsgw) and wind speed (wind). Note that htsgw does not show that much results for the last31days duration because there was no matched pair for the last week of September ~ current for this wave field. You can check some saved images in the images folder in "old" directory for duration of mid-September when we had more matched pairs.

**Review by AndrewBenjamin-NOAA — APPROVED** [review]
After looking at the code and reviewing the subsequent plots, I approve these changes. Thank you for pushing these changes to your test site, it's nice to see the end result of all this good work.

**Comment by malloryprow** [comment]
@SamiraArdani-NOAA Can you do a sync fork before I merge?

**Pull request merged**
